addons/website_forum/i18n/website_forum.pot

```diff
@@ -2991,6 +2991,12 @@ msgstr ""
 #. module: website_forum
 #. odoo-javascript
 #: code:addons/website_forum/static/src/js/website_forum.js:0
```

Comment:
in the commit message
> Sorry, you cannot vote for your own post**s**

Reply:
Hello @std-odoo 

I have made the changes :+1: